examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,4210 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    r = 2.0 * (np.random.rand(ny, nx) - 0.5) * pert # create random number perturbations
</pre><p>Probably a good idea to set a seed here for reproducibility</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>You're right, I should've thought about it.</p>

Reply:
Will definitely need doing for the notebook CI to pass